### PR TITLE
added longest streak + lobby K/D

### PR DESCRIPTION
**Pull request by Haroon96**

### CODE REVIEW

File: src/cod/stats.ts [hunk 1]
```diff
@@ -87,23 +87,30 @@ function sum(stats, field): number {
...
+    const stats = matches.map(x => x.segments[0].stats);
+    const statValues: Stats = {
         'Matches': stats.length,
         'Kills': sum(stats, 'kills'),
         'Deaths': sum(stats, 'deaths'),
         'Assists': sum(stats, 'assists'),
         'Time Played': formatDuration(sum(stats, 'timePlayed')),
         'Avg. Game Time': formatDuration(sum(stats, 'timePlayed') / stats.length),
-        'Avg. Team Placement': parseInt((sum(stats, 'teamPlacement') / Math.max(stats.length, 1)).toString()),
+        'Avg. Team Placement': Math.floor(sum(stats, 'teamPlacement') / stats.length),
```
Comment (osama-h-rana) [inline src/cod/stats.ts:98]: We should probably convert this to a percentile as the team placement number doesnt make sense when multiple modes (solos-quads) are played within the specified time frame.

So if you placed 100/150 in solos (33rd percentile) and 20/50 in trios (60th percentile), your overall percentile would be 46.5 percentile. 

I hope my percentile math isn't all wrong 😅

File: src/cod/stats.ts [hunk 2]
```diff
@@ -87,23 +87,30 @@ function sum(stats, field): number {
...
         'Headshots': sum(stats, 'headshots'),
         'Executions': sum(stats, 'executions'),
+        'Longest Streak': Math.max(...stats.map(x => x.longestStreak ? x.longestStreak.value : 0)),
```
Comment (osama-h-rana) [inline src/cod/stats.ts:101]: What is longest streak?

**Review by osama-h-rana — CHANGES_REQUESTED** [review]

File: src/cod/stats.ts (hunk 1 above)
Comment (Haroon96) [reply src/cod/stats.ts:98]: PR has nothing to do with the stat itself and only fixes some bad code related to int parsing. We can revisit this in a different issue.

File: src/cod/stats.ts (hunk 2 above)
Comment (Haroon96) [reply src/cod/stats.ts:101]: # of consecutive kills before dying.

**Review by osama-h-rana — APPROVED** [review]

**Pull request merged**